### PR TITLE
Oi/#8 dual js support

### PR DESCRIPTION
**Pull request by greenermoose**
We fixed the hardware so all drive module offsets should be zero now.

### CODE REVIEW

**Review by BenBernardCIS — CHANGES_REQUESTED** [review]
Version.java should not be committed...you may need to refresh this branch from main to get the latest git.ignore

File: src/main/java/org/frc5687/rapidreact/OI.java [hunk 1]
```diff
@@ -24,10 +32,11 @@ public OI() {
 
         _translation = new Joystick(0);
         _rotation = new Joystick(1);
+        resetNavX = new JoystickButton(_translation, 5);
     }
 
     public void initializeButtons(DriveTrain driveTrain) {
-        //There's nothing to init here
+        resetNavX.whenPressed(new ResetNavX(driveTrain));
```
Comment (deslobodzian) [inline src/main/java/org/frc5687/rapidreact/OI.java:39]: Do we need a command for this? If not, we can change this to `restNavX.whenPressed(driveTrain::resetNavX)` thanks to the help of the reference operator `::`. They are similar to lambda operators `->`.

File: src/main/java/org/frc5687/rapidreact/subsystems/DriveTrain.java [hunk 2]
```diff
@@ -134,7 +137,7 @@ public void updateDashboard() {
 
         metric("BR/Encoder Wheel Vel", _backRight.getWheelVelocity());
         metric("BR/Predicted Wheel Vel", _backRight.getPredictedWheelVelocity());
-
+        DriverStation.reportWarning("Odometry Pose" + getOdometryPose().toString(), false);
```
Comment (deslobodzian) [inline src/main/java/org/frc5687/rapidreact/subsystems/DriveTrain.java:140]: Again, you can shorten this to just `error("Message in here")` thanks to our Outliers subsystem abstract class.

File: src/main/java/org/frc5687/rapidreact/Constants.java [hunk 3]
```diff
@@ -28,16 +28,12 @@
...
         public static final double DEADBAND = 0.2;
 
-        public static final double MAX_MPS = 3.5; // Max speed of robot (m/s) .
+        public static final double MAX_MPS = 1.5; // Max speed of robot (m/s) .
 
-<<<<<<< Updated upstream
-        public static final double MAX_ANG_VEL =
-                Math.PI * 2.0; // Max rotation rate of robot (rads/s)
+        public static final double MAX_ANG_VEL = Math.PI * 1.5; // Max rotation rate of robot (rads/s)
         public static final double MAX_MPSS = 0.5; // Max acceleration of robot (m/s^2)
```
Comment (deslobodzian) [inline src/main/java/org/frc5687/rapidreact/Constants.java:34]: Only affects auto, not teleop.

**Comment by deslobodzian** [comment]
@BenBernardCIS It seems for gitignore to not continuously commit the Version.java you need to remove it from the git tracked files (I think).

File: src/main/java/org/frc5687/rapidreact/Constants.java [hunk 4]
```diff
@@ -18,7 +18,7 @@
         //Swerve modules renamed from positions to directions
         //I did it ya'll happy? :) Where's my golden star?
         public static final Translation2d NORTH_EAST = new Translation2d(WIDTH / 2.0, LENGTH / 2.0);
-        public static final double NORTH_EAST_OFFSET = 0; // radians
+        public static final double NORTH_EAST_OFFSET = -0.349066; // radians
```
Comment (deslobodzian) [inline src/main/java/org/frc5687/rapidreact/Constants.java:21]: Hardware was set to have an offset of 0, why is this changed?

File: src/main/java/org/frc5687/rapidreact/Constants.java (hunk 3 above)
Comment (KilroyWasHere-cs-j) [reply src/main/java/org/frc5687/rapidreact/Constants.java:34]: Can we find a way to be able to adjust the accel for teleop? If it's possible during auto it should be possible during teleop.

**Comment by KilroyWasHere-cs-j** [comment]
Changes made.
